docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,286 @@
+contributing to SpECTRE, we strongly encourage you to contact us at
+questions@spectre-code.org to discuss possible contributions.
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report for
+SpECTRE. Following these guidelines helps maintainers and the
+community understand your report, reproduce the behavior, and find
+related reports.
+
+Before creating bug reports, please **perform a
+[search](https://github.com/sxs-collaboration/spectre/issues)** to see
+if the problem has already been reported. If it has **and the issue is
+still open**, add a comment to the existing issue instead of opening a
+new one.
+
+> **Note:** If you find a **Closed** issue that seems like it is the
+> same thing that you're experiencing, open a new issue and include a
+> link to the original issue in the body of your new one.
+
+#### How Do I Submit A (Good) Bug Report?
+
+Bugs are tracked as [GitHub
+issues](https://guides.github.com/features/issues/). When you are
+creating a bug report, please **include as many details as
+possible**. Fill out the required template, as the information it asks
+for helps us resolve issues faster.
+
+Explain the problem and include additional details to help maintainers
+reproduce the problem:
+
+* **Use a clear and descriptive title** for the issue to identify the
+  problem.
+* **Describe the exact steps which reproduce the problem** in as many
+  details as possible. For example, start by explaining how you
+  started SpECTRE, e.g. the exact command you used in the terminal, or
+  the contents of the batch script you used.
+* **Describe the behavior you observed after following the steps** and
+  point out what exactly is the problem with that behavior.
+* **Explain which behavior you expected to see instead and why.**
+
+Provide more context by answering these questions:
+
+* **Can you reproduce the problem in both debug and release mode?**
+* **Did the problem start happening recently** (e.g. after updating to
+  a new version/commit of SpECTRE) or was this always a problem?
+* If the problem started happening recently, **can you reproduce the
+  problem in an older version/commit?** What's the most recent
+  version/commit in which the problem doesn't happen?
+* **Can you reliably reproduce the issue?** If not, provide details
+  about how often the problem happens and under which conditions it
+  normally happens.
+* **Can you reproduce the problem on another machine?**
+
+Include details about your configuration and environment:
+
+* **Add as an attachment** (or add the contents of) the following:
+  - The text output by SpECTRE (including any stack trace)
+  - The input file(s)
+  - $SPECTRE_BUILD_DIR/LibraryVersions.txt
+* **What is the name and version of the OS you're using**?
+* If possible (for SXS computers or HPC systems), a **path to a run
+  directory** that is accessible by SpECTRE core developers.
+
```

Comment:
I suggest we suggest that users attempt to reproduce in docker as another convenient alternative, since that's a known environment to us